### DISCUSSION TITLE
Conditional for arbitrary CSS modification?

### DISCUSSION THREAD

**Discussion by cfepdan** [Budibase/budibase#5334] [Ideas]
I understand the wisdom of not providing unrestricted access to the DOM...

But what if conditionals could be used to apply / unapply arbitrary CSS? Currently, you can affect a limited amount of styling with conditionals. But it would be nice to have the full suite of CSS styling available on a conditional basis.

**Comment by melohagan** [comment]
It would be awesome to have access to bindings in the 'Edit custom CSS' drawer. 

One work around you can do is to use show conditions: https://docs.budibase.com/docs/conditions
Then you can have duplicate containers  / components. One with a particular style to be shown on a condition, another with a different style to be shown on the inverse condition.
